spec/order_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../app/lib/order'
+
+describe 'an order' do
+
+  before(:each) do
+    @order = Order.new
+  end
+
+  describe 'initialising an order' do
+
+    it 'should be empty when initialised' do
+      expect(@order.order_list).to be_empty
+    end
+
+  end
+
+  describe 'adding an item to an order' do
+
+    it 'should accept an item and a quantity into an order' do
+      @order.add_item('Cafe Latte', 2)
+      expect(@order.order_list.first[:item]).to eq('Cafe Latte')
```

Comment:
Only have one expectation per test - here you could actually just test for the whole hash